.project-template/fill_template_vars.sh

```diff
@@ -4,7 +4,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-PROJECT_ROOT=$(dirname $(dirname $(readlink -f $0)))
+PROJECT_ROOT=$(dirname $(dirname $(python -c 'import os, sys; sys.stdout.write(os.path.realpath(sys.argv[1]))' "$0")))
```

Comment:
This is getting pretty unwieldy. Maybe:

```sh
THIS_ABSOLUTE_PATH=$(python -c 'import os, sys; sys.stdout.write(os.path.realpath(sys.argv[1]))' "$0")
PROJECT_ROOT=$(dirname $(dirname $THIS_ABSOLUTE_PATH))
```
Or maybe the whole script should be converted to python. It's probably complex enough that it's time. I'll add an issue.

Reply:
I think that's a good idea.  I've honestly never had good luck with `sed`.  It acts weird depending on locale settings and properly escaping `sed` commands is a nightmare.